README.adoc

```diff
@@ -323,6 +323,29 @@ foo&. bar
 foo&.bar
 ----
 
+=== Safe navigation
+
+Avoid long chains of `&.`. Replace with `.` if they introduce unnecessary conditional logic. 
+Consider other approaches such as delegation.
+
+For condition where users are guaranteed to have an address and addresses are guaranteed to have a zip code
+
+[source, thoughtbot(https://thoughtbot.com/blog/ruby-safe-navigation#conditional-equivalence)]
```

Comment:
This breaks rendering
![image](https://user-images.githubusercontent.com/6916/209480000-9b13d0e4-c009-4a8a-ba77-1093ccee9904.png)